src/main/scala/me/jeffmay/neo4j/client/Config.scala

```diff
@@ -0,0 +1,76 @@
+package me.jeffmay.neo4j.client
+
+import com.typesafe.config.Config
+
+import scala.concurrent.duration._
+import scala.util.Try
+
+case class Neo4jBasicAuth(username: String, password: String)
+object Neo4jBasicAuth {
+
+  /**
+    * The initial username and password of Neo4j that you are asked to change; just for reference.
+    */
+  final val Initial: Neo4jBasicAuth = Neo4jBasicAuth("neo4j", "neo4j")
+
+  /**
+    * You should really change these, but for testing purposes these are useful defaults.
+    */
+  final val UnsafeDefaultPassword: Neo4jBasicAuth = Neo4jBasicAuth("neo4j", "password")
+
+  def fromConfig(config: Config, path: String = Neo4jClientConfig.DefaultRoot): Neo4jBasicAuth = {
+    // TODO: Handle nulls
```

Comment:
Remove. Play 2's Java API for configs throw useful exceptions instead of returning `null`. Yay!